scanner/src/main/kotlin/Main.kt

```diff
@@ -99,6 +99,12 @@ object Main {
             order = PARAMETER_ORDER_OPTIONAL)
     private var inputPath: File? = null
 
+    @Parameter(description = "The list of scopes that shall be scanned for licenses. Works only with the " +
```

Comment:
Please remove "for licenses", we could also integrate scanners that scan for something else.